script.js

```diff
@@ -1,4 +1,169 @@
+// generate dice function
+var randomDiceNumber = function () {
+  var diceNumber = Math.ceil(Math.random() * 6);
+  return diceNumber;
+};
+
+//create certain global varriables
+var player1FinalRoll = "";
+var player2FinalRoll = "";
+//keep track of current player
+var currentPlayer = 1;
+//keep track of scores
+var player1score = 0;
+var player2score = 0;
+
+//keep track or final number
+var player1rolls = [];
+var player2rolls = [];
+
+//keep track of rolls
+var firstRoll = "";
+var secondRoll = "";
+
+//gamemode
+currentGameMode = "player one";
+
+var playerOneDice = function () {
+  firstRoll = randomDiceNumber();
+  secondRoll = randomDiceNumber();
+  currentGameMode = "1 choose order";
+  return `Welcome Player ${currentPlayer} . You rolled <br> Dice 1: ${firstRoll}<br> Dice 2: ${secondRoll}`;
+};
+
+var playerTwoDice = function () {
+  firstRoll = randomDiceNumber();
+  secondRoll = randomDiceNumber();
+  currentGameMode = "2 choose order";
+  currentPlayer = 2;
+  return `Welcome Player ${currentPlayer} . You rolled <br> Dice 1: ${firstRoll}<br> Dice 2: ${secondRoll}`;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //for them to check leaderboard
+  if (input == "score") {
+    console.log(`input`, input);
+    console.log(`chicken`);
```

Comment:
it is good to use console log during development but remove them during code submission for clarity and readability of code